### DISCUSSION TITLE
Using Github markdown files as source

### DISCUSSION THREAD

**Discussion by arky** [quarto-dev/quarto-cli#3616] [Q&A]
Wonder if its possible to parse Github markdown [README.md ](https://github.com/neocarto/bertin/blob/main/README.md) with quarto to generate PDF and HTML documentation.

**Comment by cscheid** [comment]
Did you try it? Did you run into some specific trouble? This should generally. work.

There's one caveat which is that we use `.md` files as intermediate representations so you probably need to rename the file to `.qmd` ahead of rendering.

**Reply by arky** [reply]
Thanks @cscheid  I think rendering without renaming the file worked 

`quarto render README.md --metadata title="sometitle" --to pdf`
